src/components/Text/Text.js

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable react/no-danger */
 import * as PropTypes from 'prop-types';
 import * as React from 'react';
 
 export default class Text extends React.Component {
   render() {
     let fontWeight;
+    const textRender = () => ({ __html: this.props.text.replaceAll('<br />') });
```

Comment:
Using `<br />` for line break is inconsistent with Popover `\n`used in Popover

Reply:
`.replaceAll` expects 2 parameters..

```suggestion
    const textRender = () => ({ __html: this.props.text.replaceAll('\n', '<br />') });
```